cpp/src/utils/message.hh

```diff
@@ -94,6 +109,12 @@ namespace torchserve {
     Parameters parameters;
 
     InferenceRequest() {};
```

Comment:
Is this still needed?

Reply:
yes, that' used [here](https://github.com/pytorch/serve/blob/cpp_backend/cpp/test/backends/torch_scripted/torch_scripted_backend_test.cc#L35)